include/dali/core/geom/vec.h

```diff
@@ -57,6 +57,8 @@ template <int N>
 using dvec = vec<N, double>;
 template <int N>
 using bvec = vec<N, bool>;
+template <int N>
+using fvec = vec<N, float>;
```

Comment:
Isn't `vec` already `fvec` (not that I like it)?

Reply:
I don't know

Reply:
It was rhetorical question. 
It is.